src/odemis/acq/move.py

```diff
@@ -2871,31 +2885,29 @@ def moveAbs(self, pos: Dict[str, float], **kwargs) -> Future:
                 pos[key] = self.position.value[key]
 
         # pos is a position, so absolute conversion
-        pos_stage = self.pm.from_sample_stage_to_stage_position(pos)
+        pos_stage = self._pm.from_sample_stage_to_stage_position(pos)
         logging.debug("converted absolute move from %s to %s", pos, pos_stage)
         return self._stage_bare.moveAbs(pos_stage, **kwargs)
 
     @isasync
     def moveRelChamberReferential(self, shift: Dict[str, float]) -> Future:
```

Comment:
Sorry, I found out that "referential" is not the usual term in English :see_no_evil: . It should be more like "reference frame" or "coordinate system". Let's rename that function to `moveRelChamberCoordinates`... or maybe you have a better suggestion?